homeassistant/components/triggercmd/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "token": "TRIGGERcmd token"
```

Comment:
Rather use the token common name and move the triggercmd thing to the data_description